Packs/OpenCTI/Integrations/OpenCTI/OpenCTI.py

```diff
@@ -263,8 +261,6 @@ def indicator_create_command(client: OpenCTIApiClient, args: Dict[str, str]) ->
         simple_observable_value = value
     try:
         # cti code prints to stdout so we need to catch it.
-        old_stdout = sys.stdout
```

Comment:
we're sure this is a safe chagne? Should we add a parameter for this integration as well?